assets/js/main.js

```diff
@@ -128,7 +128,7 @@ let footer = $(`
     <i class="fab fa-github"></i>
       </a>
   <br>
-<p style="color:white;><a href=" ">contact@johndoe.com</a></p>
+<p style="color:white;><a href=" " class="col-sm-12 col-md-12s">contact@johndoe.com</a></p>
```

Comment:
What's the need of giving of col-md-12 and col-sm-12 simultaneously. Acc to me If you give cd-sm-12 it will still work. One problem is a typo(it should be 12 not 12s).

Sorry I don't want to review, but i am pointing this so that @Nishant2907 will able to do it correctly. And There will no need to do it again by me. As You have also assign it to me. (Sorry : I Know Reviewing PR is out of rule of NWOC)

Thank you !!